### DISCUSSION TITLE
MathNotepad doc : typed

### DISCUSSION THREAD

**Discussion by Phil0oo** [josdejong/mathjs#2410] [Q&A] [answered]
Hello,
I am a fan of your work and use the notepad quite a few.
In the doc pages for typed function, the example is not working on my side.
To make it work I changed it to 
double = typed({ "number": f(x)=x+x })

ps: I am new to Git so I am not sure whether it is the right place to highlight such little issue.

**Comment by josdejong** [comment, marked as answer]
Thanks for reporting @Phil0oo . I've fixed the example via 46a84ff21894c9ddd41eb6086a0ea8df8782e87a (the fix is not yet published).

**Reply by gwhitney** [reply]
Long since published now, so locking this discussion.
